debian/not-installed

```diff
@@ -5,8 +5,6 @@ usr/bin/mysql_embedded # Huge 500 MB file. Not intended for distribution via any
 usr/bin/mytop # Mytop is distributed from a separate source package
 usr/lib/sysusers.d/sysusers.conf
 usr/lib/tmpfiles.d/tmpfiles.conf
-usr/lib/mysql/plugin/JavaWrappers.jar # These are only built if JNI/libjawt.so is installed from e.g. openjdk-8-jre-headless
-usr/lib/mysql/plugin/JdbcInterface.jar # These are only built if JNI/libjawt.so is installed from e.g. openjdk-8-jre-headless
```

Comment:
Actually this is not build without having 'openjdk-11-jdk-headless' on building stage dependencies. So if we just want to have pre-build git versions of Java classes then JavaWrappers.jar should be enabled.

Reply:
I don't understand this comment. What is "pre-build git version"? Git is for version control, and should have only source code. The second somebody adds a binary such as `storage/columnstore/columnstore/utils/jemalloc/libjemalloc.so.2` or `storage/connect/JavaWrappers.jar` into git, then something has conceptually gone badly wrong. It is not just malpractice, but potentially also a copyright violation.

Reply:
These JAR files contains prebuild classes so and they are used in testcase: jdbc_postgresql.result so if they are removed then PostgressSQL class should be compiled also to some jar what used in testing. They should not be installed but I don't think they should be removed from git. They should be removed with another PR. I agree with you @ottok that they should not be there is first place but that is another battle.